sessions/TypeScript-100/Module-102-common-types/common-types.md

```diff
@@ -0,0 +1,81 @@
+
+# Module 102: Common Types
+
+## Primitives
+
+Along with this file, there are code examples of each type in [common-types-examples.ts](./common-types-examples.ts).
+
+
+Similar to JavaScript, there are 3 commonly used primitives
+- `boolean`
+	- Same as JavaScript, simple true/false value
+- `number`
+    - There is no int or float, everything is number to JavaScript /TS
+- `string`
+	- Like JavaScript , strings can use single quotes or double quotes    
+	- Template string using backticks and the ${} syntax
+- `symbol `
+	- There is a primitive in JavaScript used to create a globally unique reference via the function Symbol()
+	- Symbol represenTypeScript unique tokens that can be used as keys for object properties.
+- `bigint` 
+	- Introduced in TypeScript 3.2, it provides a way to represent whole numbers larger than 253.
+	- You can get a bigint in 2 ways:
+		- Calling the BigInt() function    
+		- Writing a BigInt literal by adding `n` to the end of any numeric integer
+	- NOTE: You can use the bigint only if you are targeting version ESNext.
+- `null`
+	- Like JavaScript, null is used to indicate an absent value
+- `undefined`
+	- Like JavaScript, undefined is used to indicate an uninitialised value
+
+Note: `null` and `undefined` depend on whether you have the [strictNullChecks](https://www.typescriptlang.org/tsconfig#strictNullChecks) option on (this will be covered in a later module)
```

Comment:
```suggestion
Note: `null` and `undefined` will not be factored into any type checks unless you have the [strictNullChecks](https://www.typescriptlang.org/tsconfig#strictNullChecks) option enabled (this will be covered in a later module)
```

Worried that this could read as though you can't use `null` or `undefined` at all :)